tests/unittests/Foundation/NSPropertyListSerializationTests.mm

```diff
@@ -29,10 +29,7 @@
     });
     ASSERT_NE(nil, plistDict);
 
-    NSDateFormatter* formatter = [[NSDateFormatter new] autorelease];
-    ASSERT_NE(nil, formatter);
-    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss'Z'"];
-    NSDate* date = [formatter dateFromString:@"2014-12-15T19:48:38Z"];
+    NSDate* date = [NSDate dateWithTimeIntervalSinceReferenceDate:440365718]; // Mon Dec 15 2014 19:48:38 GMT
```

Comment:
>dateWithTimeIntervalSinceReferenceDate [](start = 27, length = 38)

nit; could use calendar [dateWithEra:1 year:2014 hour:19 minute:48 second:38] if you wanted to avoid the magic number there.